network/network_interface.ml

```diff
@@ -203,6 +204,7 @@ module Interface = struct
 	external get_capabilities : debug_info -> name:iface -> string list = ""
 	external is_connected : debug_info -> name:iface -> bool = ""
 	external is_physical : debug_info -> name:iface -> bool = ""
+	external has_vlan: debug_info -> device:string -> vlan:int64 -> bool = ""
```

Comment:
`vlan` does not need to be `int64`. The one in `bridge_config_t` uses just `int`.